tests/SharedTests/TestCreate.cs

```diff
@@ -17,6 +17,32 @@ public class TestCreate : UnitTestBase
     {
         public TestCreate(XrmMockupFixture fixture) : base(fixture) { }
 
+        [Theory]
+        [InlineData(60, true)]
+        [InlineData(-60, false)]
+        [InlineData(1, true)]
+        [InlineData(-1, false)]
+        public void TestCreateOverriddenCreatedOn(int addMinutes, bool throwExceptionExpected)
+        {
+            var dateTimeUtcNow = DateTime.UtcNow;
+            var dateTime = dateTimeUtcNow.AddMinutes(addMinutes);
+            var contact = new Contact()
+            {
+                OverriddenCreatedOn = dateTime,
+            };
+            if (throwExceptionExpected)
+            {
+                Assert.Throws<FaultException>(() => orgAdminService.Create(contact));
+            }
```

Comment:
Suggest adding check for the exception message
```suggestion
                Assert.Equal("Trying to create entity 'contact', but overriddencreatedon cannot be set to a date in the future", ex.Message);
            }
```